src/test-run/index.js

```diff
@@ -582,6 +588,11 @@ export default class TestRun extends AsyncEventEmitter {
         return this._enqueueCommand(command, callsite);
     }
 
+    _logCommand (command) {
+        this.commands.push(command);
```

Comment:
I guess we should not store all commands. We can check each command during test execution and store `browserResized` in variable.

Reply:
I thought, that it would be helpful in future. It can help us to prevent some variables like 'browserResized', 'roleUsed' or 'expressionIsExecuted'. However, since we do not have such kind of commands yet, lets make one `browserResized` flag